src/main/java/Duke.java

```diff
@@ -1,10 +1,112 @@
+import java.util.Objects;
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
         String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
+                + "|  _ \\ _   _| | _____\n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        int taskNum = 0;
+        String userCommand;
+        Task[] tasks = new Task[100];
+        Scanner userType = new Scanner(System.in);
```

Comment:
line 12-15 can be refactored into a userInput function so you can change it easier in the future!